public/script.js

```diff
@@ -0,0 +1,131 @@
+const searchInput = document.getElementById('search');
+const autocomplete = document.querySelector('.autocomplete');
+const emptyIcon = document.querySelector('.empty-icon');
+const searchIcon = document.querySelector('.search-icon');
+const cardsCont = document.querySelector('.cards-container');
+
+const fetch = (method, uri, cb) => {
```

Comment:
what if the method is POST? how would you handle sending the request body with xhr?

Reply:
We deleted it, we thought it wouldn't be necessary 😅 
We will then use the data event listener if the method is POST, and do the logic.